raiden/utils/migrations/v19_to_v20.py

```diff
@@ -0,0 +1,75 @@
+import json
+
+from raiden.storage.sqlite import SQLiteStorage
+
+SOURCE_VERSION = 19
+TARGET_VERSION = 20
+
+
+def _add_onchain_locksroot_to_channel_settled_state_changes(
+        storage: SQLiteStorage,
+) -> None:
+    """ Adds `our_onchain_locksroot` and `partner_onchain_locksroot` to
+    ContractReceiveChannelSettled. """
+    batch_size = 50
+    batch_query = storage.batch_query_state_changes(
+        batch_size=batch_size,
+        filters=[
+            ('_type', 'raiden.transfer.state_change.ContractReceiveChannelSettled'),
+        ],
+    )
+    for state_changes_batch in batch_query:
+        updated_state_changes = []
+        for state_change in state_changes_batch:
+            data = json.loads(state_change.data)
+            msg = 'v18 state changes cant contain our_onchain_locksroot'
+            assert 'our_onchain_locksroot' not in data, msg
+
+            msg = 'v18 state changes cant contain partner_onchain_locksroot'
+            assert 'partner_onchain_locksroot' not in data, msg
+
+            data['our_onchain_locksroot'] = None
+            data['partner_onchain_locksroot'] = None
```

Comment:
I opted for using `None` for locksroot values because i  cannot think of a way to "reliably" get those values. This is because the `get_participants_details` requires passing `participant1` and `participant2`... it's kinda hard to get the value for `partner` in that case. The raiden_event_handler will query the onchain locksroots again if it sees None values